src/main.c

```diff
@@ -1,4 +1,4 @@
-#include <functions.h>
+#include "../include/functions.h"
```

Comment:
We have -I for a reason

Reply:
That's true. I was just in a hurry to make something work. I'll probably fix this tonight.